testomania/src/main/java/com/earth/testomania/technical/presentation/TechnicalTestsScreen.kt

```diff
@@ -53,8 +53,10 @@ fun TechnicalTestsScreen(
     val viewModel: QuizViewModel = quizViewModel(category)
 
     val data = viewModel.data
+    val errorState = viewModel.error.collectAsState()
```

Comment:
you can use `by` instead of equal signed and then you can use `errorState` without errorState.value